guide/getting-started/visual-language.md

```diff
@@ -39,6 +39,7 @@ layout = "full-width"
 
 Since Bitcoin answers to no central authority, there is no single symbol or logo that can visually represent it, though many try. Here are some of the most recognized attempts to create a universal logo for Bitcoin.
 
+## The Bitcoin symbol
 
 ### Original symbol
```

Comment:
Maybe add the date the core client was launched with this symbol.

Reply:
Do we know this date?

Reply:
Calling this 'The Bitcoin' symbol is not helping IMO. It's a gold coin with 'BC' in the middle.
When people say the Bitcoin symbol today, they mostly refer to the Bitboy update.

Reply:
@danielnordh I added this heading because I don't like the page jumping from an H1 to an H3 heading. Let me see what I can do to amend this.

Reply:
Amended in d8b2932
@Bosch-0 I added a tidbit about the launch date
@danielnordh I removed "The Bitcoin symbol" and adjusted the heading levels.
How does that look?